data/hai missions.txt

```diff
@@ -393,6 +397,7 @@ mission "Unfettered returning home"
 		has "Unfettered Tribute 3: done"
 		random < 40
 	on offer
+		log "Helped a juvenile Hai in Unfettered space seek refuge in Hai-home. Despite the ongoing hostilities, he seems he got a pretty warm welcome."
```

Comment:
`he seems he got`
I would change it to `it seems he got` or `he seems to have gotten`. Or `he seems to have received`. (Also, typo, Derpy.)